packages/elements/src/components/ino-fab/ino-fab.tsx

```diff
@@ -122,14 +130,17 @@ export class Fab implements ComponentInterface {
       'mdc-fab--mini': this.inoMini,
     });
 
+    const iconSlotHasContent = hasSlotContent(this.el, 'ino-icon-leading');
+
     return (
       <Host id={this.uniqueHelperId}>
         <button class={classFab} disabled={this.inoDisabled}>
           <span class="material-icons mdc-fab__icon">
-            {this.inoIcon ? (
-              <ino-icon class="mdc-button__icon" ino-icon={this.inoIcon} />
-            ) : (
-              <slot></slot>
+            { this.inoIcon && !iconSlotHasContent && (
+              <ino-icon ino-icon={this.inoIcon}/>
+            )}
+            { iconSlotHasContent && (
+              <slot name="ino-icon-leading"/>
```

Comment:
Stencil introduced annotations for slots which will be used to generate a table in the readme. We should use it here as well.

See:
https://github.com/inovex/elements/blob/8b91f4e51abf5a920ccf3718ccc54e13802768f1/packages/elements/src/components/ino-button/ino-button.tsx#L16-L19

Reply:
✅